code/modules/reagents/chemistry/reagents/water.dm

```diff
@@ -383,6 +384,47 @@
 		M.AdjustCultSlur(20 SECONDS) //CUASE WHY THE HELL NOT
 	return ..() | update_flags
 
+/datum/reagent/fuel/unholywater/overdose_process(mob/living/M, severity)
+	var/multiplier = 1
+	if(!iscultist(M))
+		multiplier = 2
+
+	var/list/overdose_info = ..()
+	var/effect = overdose_info[REAGENT_OVERDOSE_EFFECT]
+	var/update_flags = overdose_info[REAGENT_OVERDOSE_FLAGS]
+
+	if(severity == 1)
+		if(effect <= 2)
+			M.visible_message("<span class='warning'>[M] starts to grab [M.p_their()] head in pain!</span>")
+			M.EyeBlurry(7 SECONDS * multiplier)
+			M.Confused(7 SECONDS * multiplier)
+
+		else if(effect <= 4)
+			M.visible_message("<span class='warning'>[M] freezes up!</span>")
+			M.Jitter(15 SECONDS * multiplier)
+			M.Stun(8 SECONDS * multiplier)
+			M.emote("gasp")
+
+		else if(effect <= 7)
+			M.AdjustCultSlur(10 SECONDS * multiplier)
```

Comment:
these three as well for swtich statement

Reply:
I agree with sirryan, switch here please